plugins/generator-1.20.4/neoforge-1.20.4/templates/livingentity/livingentity.java.ftl

```diff
@@ -414,25 +406,22 @@ public class ${name}Entity extends ${extendsClass} <#if data.ranged>implements R
 
 	private final CombinedInvWrapper combined = new CombinedInvWrapper(inventory, new EntityHandsInvWrapper(this), new EntityArmorInvWrapper(this));
 
-	@Override public <T> LazyOptional<T> getCapability(@Nonnull Capability<T> capability, @Nullable Direction side) {
-		if (this.isAlive() && capability == ForgeCapabilities.ITEM_HANDLER && side == null)
-			return LazyOptional.of(() -> combined).cast();
-
-		return super.getCapability(capability, side);
+	public CombinedInvWrapper getInventory() {
+		return combined;
 	}
 
    	@Override protected void dropEquipment() {
 		super.dropEquipment();
-		for(int i = 0; i < inventory.getSlots(); ++i) {
+		for (int i = 0; i < inventory.getSlots(); ++i) {
 			ItemStack itemstack = inventory.getStackInSlot(i);
 			if (!itemstack.isEmpty() && !EnchantmentHelper.hasVanishingCurse(itemstack)) {
 				this.spawnAtLocation(itemstack);
 			}
 		}
 	}
-    </#if>
+	</#if>
 
-	<#if data.entityDataEntries?has_content || (data.guiBoundTo?has_content && data.guiBoundTo != "<NONE>")>

```

Comment:
I feel this should not be removed

Reply:
:heavy_check_mark: Fixed

Reply:
Wait, why it shows the commit but does not update the files tab?

Reply:
Please retest affected parts